### ISSUE TITLE
When pressing the Enter key on the "Create blob container" menu item, NVDA repeatedly announces "resource tree, use navigation keys to navigate":A11y_Azure Tools Storage Explorer_Creating a blob container_Screen Reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8373]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows11

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #Screenreader; #WCAG1.3.1; #FTP; #NVDA; #Jaws; #STP; #Closed; #Regressed:02-12-25;

## Environment Details:
Application: Microsoft azure storage explorer
Version: 1.36.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
NVDA Version: 2024.4.1
JAWS Version: 2024.2312.53

## Repro steps

1. Turn on NVDA
2. Launch the Azure Tools Storage Explorer.
3. TAB till "Blob container" in tree view and open context menu 
4. Pree down arrow key to "create blob container" menu item and press ENTER key
5. Observe that when pressing the Enter key on the "Create blob container" menu item, NVDA repeatedly announces "resource tree, use navigation keys to navigate"

## Actual Experience

When pressing the Enter key on the "Create blob container" menu item, NVDA repeatedly announces "resource tree, use navigation keys to navigate"
Note: Issue is repro with JAWS also

## Expected Experience

When pressing the Enter key on the "Create blob container" menu item, NVDA should not announce repeatedly "resource tree, use navigation keys to navigate".

## User Impact:
Screen reader use will face difficulty if screen reader announces same information multiple time.

## Attachment
NVDA
![Image](https://github.com/user-attachments/assets/6dc40596-6c07-4d08-92ea-c496dd284e58)

Jaws
https://github.com/user-attachments/assets/f8d3913e-90fc-4c96-9d1f-042880ce18ca

**Comment by InduPriya1805** [comment]
Rev:inse

**Comment by craxal** [comment]
@kapilvaishna Fix is ready for validation.

**Comment by kapilvaishna** [comment]
@craxal Issue is repro in build 1.37.0 (104). PFA for reference.

![Image](https://github.com/user-attachments/assets/db616b7f-8a32-45b9-8100-a2cf89dfe9ad)

**Comment by craxal** [comment]
@kapilvaishna Sorry, I forgot to send you a link to the new build. The build number you should have should be at least 20250128.2 or later. In the future, can you please report the build number that you test on?

**Comment by craxal** [comment]
@kapilvaishna Have you been able to verify whether this is fixed yet?

**Comment by kapilvaishna** [comment]
@craxal sorry for delay I was on vacation for one week so did not verify bugs. today i will verify all resolved bug.


Issue is not repro in below version so closing the bug.
Version 1.38.0 -dev (104)

![Image](https://github.com/user-attachments/assets/8a52cd96-1246-4c0f-a472-e55595a21413)
